src/braket/pennylane_plugin/braket_device.py

```diff
@@ -95,16 +98,21 @@ def __init__(
         device: Device,
         *,
         shots: Union[int, None],
+        noise_model: Optional[NoiseModel] = None,
         **run_kwargs,
     ):
         super().__init__(wires, shots=shots or None)
         self._device = device
         self._circuit = None
         self._task = None
+        self._noise_model = noise_model
         self._run_kwargs = run_kwargs
         self._supported_ops = supported_operations(self._device)
         self._check_supported_result_types()
 
+        if noise_model and self._device.name not in ["DensityMatrixSimulator", "dm1"]:
```

Comment:
Should use device properties to extract supported operations rather than the name of the device.

Reply:
sounds good, fixed!

Reply:
@speller26 I updated the validation, and it's ready for review, thanks!